### ISSUE TITLE
Scss variable name triggers color picker

### ISSUE THREAD

**Issue by jwheatp** [easylogic/codemirror-colorpicker#46]
First, thank you for your great work.

**Describe the bug**
When using CodeMirror in Scss mode with variables, this happen if you choose a color name as variable name :

![image](https://user-images.githubusercontent.com/7983490/145558574-29c90dc5-93e6-4f00-b036-ff8d079bf5e3.png)

Seems we need to add to the exluded token the "variable" token.
https://github.com/easylogic/codemirror-colorpicker/blob/main/src/extension/codemirror/colorview.js#L7

![image](https://user-images.githubusercontent.com/7983490/145559048-eb2c51e5-0b6d-49dd-9bf8-67dfff2a4e86.png)

Is it enough ? :)

**Comment by jwheatp** [comment]
The mode is "text/x-scss"


```js
CodeMirror(this.$refs.editor, {
      value: this.code,
      mode: "text/x-scss",
      theme: "ambiance",
      colorpicker : {
          mode : 'edit',
          type : 'vscode'
      },
    });
```

**Comment by easylogic** [comment]
@jwheatp  

I released new version (codemirror-colorpicker@1.9.73) 

Please test it .

Thank you.

**Comment by jwheatp** [comment]
it works great, thank for you quick fix @easylogic !
